app/build.gradle

```diff
@@ -52,29 +57,28 @@ dependencies {
     implementation('com.crashlytics.sdk.android:crashlytics:2.9.4@aar') {
         transitive = true
     }
-    implementation 'com.android.support:design:27.1.1'
-    implementation 'com.android.support:appcompat-v7:27.1.1'
-    implementation 'com.android.support:recyclerview-v7:27.1.1'
-    implementation 'com.android.support:cardview-v7:27.1.1'
-    implementation 'com.android.support:palette-v7:27.1.1'
-    implementation 'com.android.support.constraint:constraint-layout:1.1.3'
-    implementation 'com.squareup.okhttp:okhttp:2.4.0'
-    implementation 'com.squareup.okhttp:okhttp-ws:2.4.0'
-    implementation 'org.jetbrains.anko:anko-commons:0.10.4'
-    implementation 'org.jsoup:jsoup:1.10.2'
-    implementation 'com.fasterxml.jackson.core:jackson-databind:2.9.7'
-    implementation 'org.unbescape:unbescape:1.1.1.RELEASE'
-    implementation 'com.orhanobut:dialogplus:1.11@aar'
-    implementation "org.jetbrains.kotlin:kotlin-stdlib-jdk7:$kotlin_version"
-    implementation 'me.saket:better-link-movement-method:1.2'
-    implementation 'com.github.gabrielemariotti.changeloglib:changelog:2.1.0'
-    implementation 'com.github.bumptech.glide:glide:4.7.1'
-    kapt 'com.github.bumptech.glide:compiler:4.7.1'
 
-    // Testing
-    testImplementation "org.jetbrains.kotlin:kotlin-test-junit:$kotlin_version"
-    testImplementation "com.nhaarman.mockitokotlin2:mockito-kotlin:2.0.0-RC2"
-    testImplementation 'com.squareup.okhttp:mockwebserver:2.4.0'
+    implementation "com.android.support:design:$rootProject.ext.supportLibrary"
```

Comment:
Is it really necessary to use this syntax? `$rootProject.ext.foo`? I believe you can simplify that to `$foo` instead.

For example: `implementation "com.android.support:design:$supportLibrary"`

Reply:
let me try.

Reply:
works. let me refactor this.

Reply:
done. i refactored it as you suggested.